### PR TITLE
SW-2696 setup ci

### PR DESCRIPTION
**Pull request by ShrwdFlrst**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+          command: docker-compose run --rm test
+      - store_test_results:
+          path: build
+
+  build_notify_queue_consumer:
+    executor: notify_queue_consumer/python
+    parameters:
+      container_repo_url:
+        type: string
+        default: 311462405659.dkr.ecr.eu-west-1.amazonaws.com/notify_queue_consumer
```
Comment (TomTucka) [inline .circleci/config.yml:58]: `notify_queue_consumer` should be `notify-queue-consumer`

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+          command: |
+            export SHORT_HASH=${CIRCLE_SHA1:0:7} >> $BASH_ENV
+            docker tag notify_queue_consumer:latest << parameters.container_repo_url >>:latest
+            docker tag notify_queue_consumer:latest << parameters.container_repo_url >>:$CIRCLE_BRANCH-$SHORT_HASH
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+              # We want all of the tags pushed
+              docker push << parameters.container_repo_url >>
+            else
+              docker push << parameters.container_repo_url >>:$CIRCLE_BRANCH-$SHORT_HASH
+            fi
```
Comment (TomTucka) [inline .circleci/config.yml:80]: Can you add a blank line at the end 🙂

File: .circleci/config.yml (hunk 1 above)
Comment (ShrwdFlrst) [reply .circleci/config.yml]: Done

File: .circleci/config.yml (hunk 2 above)
Comment (ShrwdFlrst) [reply .circleci/config.yml]: Done

**Pull request merged**